### ISSUE TITLE
[Feature] Disable source tables in transaction_details model

### ISSUE THREAD

**Issue by calder-holt** [fivetran/dbt_netsuite#45]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

Similar to the way you can disable features in the jira package it would be nice to be able to disable source and staging models that flow into the transaction details model if you are not using that functionality in netsuite. We are not using the `vendorcategory` or `accountingbooksubsidiary` sources so they do not flow through our Fivetran connector, and we cannot use the package. If we could disable the sources flowing into the model the same way you can [here](https://github.com/fivetran/dbt_jira_source#disabling-models), we could use the models and test them out.  

### Describe alternatives you've considered

Rewriting our own transaction details model without the sources we do not use. 

### Are you interested in contributing this feature?

- [ ] Yes.
- [ ] Yes, but I will need assistance and will schedule time during your [office hours](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) for guidance.
- [X] No.

### Anything else?

https://github.com/fivetran/dbt_jira_source#disabling-models

**Comment by fivetran-jamie** [comment]
hey @calder-holt working on this rn -- seems straightforward enough, but i'm curious about your use case, specifically around `accountingbooksubsidiary`. do y'all not use that functionality in Netsuite?

**Comment by fivetran-jamie** [comment]
also i've got a working branch if you'd like to try it out!
```yml
packages:
- git: https://github.com/fivetran/dbt_netsuite.git
  revision: feature/disable-vc-abs
  warn-unpinned: false
```

**Comment by calder-holt** [comment]
Thanks @fivetran-jamie! We actually encountered a handful of areas in the package that are features we don't use in Netsuite. We weren't using accountingbook, accountingbooksubsidiary, and vendorcategory because we don't use multi-book accounting.

**Comment by fivetran-jamie** [comment]
Ah i see -- does that mean you don't have an `accountingbook` source table either? 

Also are vendor categories a multi-book accounting thing as well? Or just another Netsuite feature y'all don't use?

**Comment by calder-holt** [comment]
Correct we also don't have an accountingbook table. I believe we don't have the functionality for vendorcategory (the table does not sync)

**Comment by fivetran-jamie** [comment]
gotcha, so are you seeing errors around `accountingbook` models as well? i'm wondering if we can consolidate this all into a `using_multibook_accounting` variable that would disable all the necessary tables (hence my vendor category question)

**Comment by calder-holt** [comment]
I think that would be a good plan

**Comment by fivetran-jamie** [comment]
this fix is in v0.6.2! there's a `netsuite2__multibook_accounting_enabled` var and `netsuite2__using_vendor_categories` var that are both true by default
